src/Symfony/Component/Config/composer.json

```diff
@@ -17,7 +17,8 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/filesystem": "~2.8|~3.0"
+        "symfony/filesystem": "~2.8|~3.0",
+        "symfony/dependency-injection": "~3.2"
```

Comment:
Ok I wasn't sure that it's more important than broad compatibility. Fixed it for existing passes in https://github.com/symfony/symfony/pull/21779.

> We can skip tests when the trait is not available

Adding the conflict avoids the need for skipping tests.